src/extension.js

```diff
@@ -470,7 +483,7 @@ async function loadCTags(tagFilePath)
 		var liner = new LineByLine(tagFilePath);
 	}
 	catch(err) {
-		vscode.window.showErrorMessage("Can not load file '.ctags'");
+		vscode.window.showErrorMessage("Can't load CTags file");
```

Comment:
Maybe then change it to something like "Can't load file ${tagFilePath} CTags!"?

Reply:
fixed